fbchat/client.py

```diff
@@ -1066,7 +1389,7 @@ def addUsersToGroup(self, user_ids, thread_id=None):
         data['action_type'] = 'ma-type:log-message'
         data['log_message_type'] = 'log:subscribe'
 
-        if type(user_ids) is not list:
+        if not isinstance(user_ids, list):
```

Comment:
As we talked about, it'd be nice to have a helper method for this

Reply:
Done